Update account owned tokens (#18684)
cloudflare/cloudflare-docs: src/content/partials/fundamentals/account-owned-tokens.mdx

---

While user tokens act on behalf of a particular user and inherit a subset of that user's permissions, account owned tokens allow you to set up durable integrations that can act as service principals with their own specific set of permissions. This approach is ideal for scenarios like CI/CD, or building integrations with external services like SEIMs where it is important that the integration continues working, even long after the user who configured the integration may have left your organization altogether. User tokens are better for ad hoc tasks like scripting, where acting as the user is ideal and durability is less of a concern. 

## Create an account owned token
:::note
Creating an account owned token requires Super Administrator permission on the account
:::

1. Log into the [Cloudflare dashboard](https://dash.cloudflare.com).
2. Go to **Manage Account** >  **Account API Tokens**.
4. Select **Create Token** and fill in the token name, permissions, and the optional expiration date for the token. 
5. Select **Continue to summary** and review the details.
6. Select **Create Token**.

> Alternatively, you can create a token using the [account owned token creation API](/api-next/resources/accounts/subresources/tokens/methods/create/).

Reviewer: ```suggestion
Alternatively, you can create a token using the [account owned token creation API](/api/resources/accounts/subresources/tokens/methods/create/).
```

---

> While user tokens act on behalf of a particular user, and inherit a subset of that user's permissions, account owned tokens allow you to set up durable integrations that can act as service principals, effectively acting as themselves with their own specific set of permissions. This approach is ideal for scenarios like CI/CD, or building integrations with external services like SEIMs where it's important that the integration keeps working, even long after the user who configured the integration may have left your organization altogether. User tokens are better for ad hoc tasks like scripting, where acting as the user is ideal, and durability is less of a concern. 

Reviewer: ```suggestion
While user tokens act on behalf of a particular user and inherit a subset of that user's permissions, account owned tokens allow you to set up durable integrations that can act as service principals with their own specific set of permissions. This approach is ideal for scenarios like CI/CD, or building integrations with external services like SEIMs where it is important that the integration continues working, even long after the user who configured the integration may have left your organization altogether. User tokens are better for ad hoc tasks like scripting, where acting as the user is ideal and durability is less of a concern. 
```

> ## Creating an account owned token

Reviewer: ```suggestion
## Create an account owned token
```

:::note

> Creating an account owned token requires Super Administrator permission on the account

Reviewer: ```suggestion
The Super Administrator permission is required to create an account owned token on your account.
```

Author: @patriciasantaana isn't this one passive voice? (I had a former boss that was a content editor, so it's been beaten into me to avoid it)

Reviewer: I think I was mainly avoiding starting the sentence with a verb. Totally get what you mean to avoid passive voice though

:::

> 1. Log into the [Cloudflare dashboard](https://dash.cloudflare.com)
> 2. In the sidebar, choose **Manage Account**
> 3. Choose **Account API Tokens**
> 4. Click **Create Token**
> 5. Navigate through the subsequent pages to set the name, permissions, and the (optional) expiration date for the token. Click **Continue to Summary**
> 6. Review the details, and finally click **Create Token**

Reviewer: These revised steps are based off of the flow that I can currently see on my dashboard -- are there upcoming updates that I'm not seeing that cause for the slight differences in labels or steps? @jhutchings1 

```suggestion
1. Log into the [Cloudflare dashboard](https://dash.cloudflare.com).
2. Go to **My Profile** >  **API Tokens**.
4. Select **Create Token** and **Create custom token**.
5. Fill in the token name, permissions, and the optional expiration date for the token. 
6. Select **Continue to summary** and review the details.
7. Select **Create Token**.
```

Author: > 2. Go to **My Profile** > **API Tokens**.

That is the correct flow for user tokens, but the entry point is different for account owned tokens, matching what I described earlier. You have to be a Super Admin of the account to see this. Let me know if you want me to invite you to an account to see it if you can't.

Reviewer: @jhutchings1 revised the wording to fit the steps you initially had. Let me know if I missed something!

> You can alternatively create a token using the [account owned token creation API](/api-next/resources/accounts/subresources/tokens/methods/create/).

Reviewer: ```suggestion
Alternatively, you can create a token using the [account owned token creation API](/api-next/resources/accounts/subresources/tokens/methods/create/).
```

Author: Does it work for you if it's fully qualified? The linter complained at me for having the full URL, but this content is part of a microsite that's not part of the repo. This loads on my end. https://developers.cloudflare.com/api-next/resources/accounts/subresources/subscriptions/methods/delete/

## Compatibility matrix

> Account owned tokens are generally available in all accounts. Some services may not support account owned tokens yet. Please see the compatibility matrix below for the latest status. 

Reviewer: ```suggestion
Account owned tokens are generally available for all accounts. Some services may not support account owned tokens yet. Refer to the compatibility matrix below for the latest status. 
```

| Product | Compatibility |

> | :---- | :---- |

Reviewer: ```suggestion
| --- | --- |
```
